Author: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Description: Fix some typos in the code

Credit to Lintian

--- a/pjsip-apps/src/pjsua/pjsua_app_cli.c
+++ b/pjsip-apps/src/pjsua/pjsua_app_cli.c
@@ -1685,7 +1685,7 @@ static pj_status_t cmd_transfer_call(pj_
 	pjsua_call_get_info(current_call, &ci);
 	pj_ansi_snprintf(out_str, 
 			 sizeof(out_str), 
-			 "Transfering current call [%d] %.*s\n",
+			 "Transferring current call [%d] %.*s\n",
 			 current_call,
 			 (int)ci.remote_info.slen, 
 			 ci.remote_info.ptr);
--- a/pjsip-apps/src/pjsua/pjsua_app_legacy.c
+++ b/pjsip-apps/src/pjsua/pjsua_app_legacy.c
@@ -1068,7 +1068,7 @@ static void ui_call_transfer(pj_bool_t n
 	pjsua_msg_data msg_data;
 
 	pjsua_call_get_info(current_call, &ci);
-	printf("Transfering current call [%d] %.*s\n", current_call,
+	printf("Transferring current call [%d] %.*s\n", current_call,
 	       (int)ci.remote_info.slen, ci.remote_info.ptr);
 
 	ui_input_url("Transfer to URL", buf, sizeof(buf), &result);
--- a/pjsip-apps/src/py_pjsua/pjsua_app.py
+++ b/pjsip-apps/src/py_pjsua/pjsua_app.py
@@ -411,7 +411,7 @@ def xfer_call():
 	else:
 		call = g_current_call		
 		ci = py_pjsua.call_get_info(g_current_call)
-		print "Transfering current call ["+ `g_current_call` + "] " + ci.remote_info
+		print "Transferring current call ["+ `g_current_call` + "] " + ci.remote_info
 		print "Enter sip url : "
 		url = sys.stdin.readline()
 		if url == "\n": 
--- a/pjsip/src/pjsua-lib/pjsua_call.c
+++ b/pjsip/src/pjsua-lib/pjsua_call.c
@@ -2540,7 +2540,7 @@ PJ_DEF(pj_status_t) pjsua_call_xfer( pjs
     PJ_ASSERT_RETURN(call_id>=0 && call_id<(int)pjsua_var.ua_cfg.max_calls &&
                      dest, PJ_EINVAL);
     
-    PJ_LOG(4,(THIS_FILE, "Transfering call %d to %.*s", call_id,
+    PJ_LOG(4,(THIS_FILE, "Transferring call %d to %.*s", call_id,
 			 (int)dest->slen, dest->ptr));
     pj_log_push_indent();
 
@@ -2621,7 +2621,7 @@ PJ_DEF(pj_status_t) pjsua_call_xfer_repl
 		      dest_call_id<(int)pjsua_var.ua_cfg.max_calls,
 		     PJ_EINVAL);
     
-    PJ_LOG(4,(THIS_FILE, "Transfering call %d replacing with call %d",
+    PJ_LOG(4,(THIS_FILE, "Transferring call %d replacing with call %d",
 			 call_id, dest_call_id));
     pj_log_push_indent();
 
